Rover/crash_check.cpp

```diff
@@ -30,7 +30,8 @@ void Rover::crash_check()
     if (!is_balancebot()) {
         if (!crashed && ((ahrs.groundspeed() >= CRASH_CHECK_VEL_MIN) ||        // Check velocity
             (fabsf(ahrs.get_gyro().z) >= CRASH_CHECK_VEL_MIN) ||  // Check turn speed
-            (fabsf(g2.motors.get_throttle()) < CRASH_CHECK_THROTTLE_MIN))) {
+            (fabsf(g2.motors.get_throttle()) < CRASH_CHECK_THROTTLE_MIN) ||
+             g2.sailboat.in_irons())) {
```

Comment:
Why?

Reply:
Sailboat In Irons all the conditions to not be crashed may be false AFAICS
* groundspeed > 0  --> probably false
* turning at some reasonable rate -> probably false
* throttle  < 5%  --> maybe false ( throttle is ignored as we have separate mainsail control)

Reply:
So the goal is to prevent detecting a crash if in iorns?

Reply:
A crash will still be detected if the CRASH_ANGLE > zero and pitch or roll has exceeded it I think.

Reply:
> A crash will still be detected if the CRASH_ANGLE > zero and pitch or roll has exceeded it I think.

Tested and verified in the simulator

Reply:
The other conditions could all also be true it there was just no wind. Maybe we would be better to default the crash check to disable for sailboats. Or add a new option to do the angle check only.

That it to say, if we want to improve the crash check for the sailboat case I would rather do it more completely. We could use one of the new option bits. "Disable moment based crash check when under sail" sort of thing.